tests/test_safe_operator.py

```diff
@@ -246,17 +263,42 @@ def test_send_ether(self):
         self.assertTrue(safe_operator.send_ether(random_address, value))
         self.assertEqual(self.ethereum_client.get_balance(random_address), value)
 
-    @lru_cache(maxsize=None)
-    def _deploy_l2_migration_contract(self) -> ChecksumAddress:
-        # Deploy L2 migration contract
-        safe_to_l2_migration_contract = self.w3.eth.contract(
-            abi=safe_to_l2_migration["abi"], bytecode=safe_to_l2_migration["bytecode"]
+    @mock.patch.object(
+        SafeOperator, "last_default_fallback_handler_address", new_callable=PropertyMock
+    )
+    @mock.patch.object(
+        SafeOperator, "last_safe_contract_address", new_callable=PropertyMock
+    )
+    def test_update_version(
+        self,
+        last_safe_contract_address_mock: PropertyMock,
+        last_default_fallback_handler_address: PropertyMock,
+    ):
+        last_safe_contract_address_mock.return_value = self.safe_contract_V1_4_1.address
+        last_default_fallback_handler_address.return_value = (
+            self.compatibility_fallback_handler.address
         )
-        tx_hash = safe_to_l2_migration_contract.constructor().transact(
-            {"from": self.ethereum_test_account.address}
+
+        safe_operator_v130 = self.setup_operator(version="1.3.0")
+        with self.assertRaises(SafeVersionNotSupportedException):
+            safe_operator_v130.update_version()
+
+        safe_operator_v111 = self.setup_operator(version="1.1.1")
+        with mock.patch.object(
+            MultiSend,
+            "MULTISEND_CALL_ONLY_ADDRESSES",
+            [self.multi_send_contract.address],
+        ):
+            safe_operator_v111.update_version()
+
+        self.assertEqual(
+            safe_operator_v111.safe.retrieve_master_copy_address(),
+            last_safe_contract_address_mock.return_value,
```

Comment:
last_safe_contract_address_mock.return_value is safe_contract_V1_4_1.address, then the v1.1.1 was updated to 1.4.1 this is possible?

Reply:
Yes